pkg/controller/disruption/disruption.go

```diff
@@ -627,8 +627,9 @@ func (dc *DisruptionController) getExpectedScale(pdb *policy.PodDisruptionBudget
 	// A mapping from controllers to their scale.
 	controllerScale := map[types.UID]int32{}
 
-	// 1. Find the controller for each pod.  If any pod has 0 controllers,
+	// Find the controller for each pod.  If any pod has 0 controllers,
 	// that's an error. With ControllerRef, a pod can only have 1 controller.
+	expectedCount = 0
```

Comment:
Nit: Unrelated to this diff, but is there a reason this function uses named return variables? Imho, it makes the code less readable. If we do move forward with this diff, could I trouble you to refactor to just use a normal return?